frontend/src/hooks/useSearch.ts

```diff
@@ -12,32 +12,37 @@ type IUseSearchOutput<T extends any> = {
     getSearchContext: () => IGetSearchContextOutput<T>;
 };
 
+const normalizeSearchValue = (value: string) =>
```

Comment:
we normalize search and filter data before any processing. This normalization allows to add spaces around commas